internal/kv/etcd/etcd_kv_test.go

```diff
@@ -909,6 +910,94 @@ func TestEtcdKV_Load(te *testing.T) {
 	})
 }
 
+func Test_WalkWithPagination(t *testing.T) {
```

Comment:
test the pagination with 1, it's not working

Reply:
I also verify it and it works